letsencrypt/cli.py

```diff
@@ -159,7 +163,34 @@ def _init_le_client(args, config, authenticator, installer):
     return client.Client(config, acc, authenticator, installer, acme=acme)
 
 
-def run(args, config, plugins):
+def _find_duplicative_certs(domains, config, renew_config):
+    """Find existing certs that duplicate the request."""
+
+    identical_names_cert, subset_names_cert = None, None
+
+    configs_dir = renew_config.renewal_configs_dir
+    for renewal_file in os.listdir(configs_dir):
+        full_path = os.path.join(configs_dir, renewal_file)
+        rc_config = configobj.ConfigObj(renew_config.renewer_config_file)
+        rc_config.merge(configobj.ConfigObj(full_path))
+        rc_config.filename = full_path
```

Comment:
I'm still extremely confused about what the correct way to initialize the various configuration objects should be. I don't think we've solved this (in several places) and I'd encourage a "configuration object hunt" or "configuration object review". (I know what this one does and I think it's probably appropriate here, but I don't fully understand what some of the other ones do.)